todos/src/App.tsx

```diff
@@ -0,0 +1,67 @@
+import { useState, useRef } from 'react';
+import './App.css';
+import {OptionBar} from './Components/OptionBar';
+import TodosList from './Components/TodosList';
+import {todo} from './Types/todo'
+
+export function App() {
+
+  function useTodos(){
```

Comment:
pls look at more examples. Hooks should not be defined inside components, but rather outside. 
Hooks are meant to be plugins. Potentially reusable pieces of functionality.